stock_invoice_picking/README.rst

```diff
@@ -0,0 +1,29 @@
+Invoice picking
```

Comment:
Please add License badge as in template: 
https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst